recipe_box/src/main/java/com/spr/recipe_box/RecipeDBRestController.java

```diff
@@ -15,32 +15,33 @@
 @RequestMapping(value = "/recipes", produces = "application/json")
 
 public class RecipeDBRestController extends RestClass {
+    private static final String RECIPES = "/recipes/";
```

Comment:
This is soooo nitpicky, but any reason this has to be all caps?

Reply:
It's the general naming convention for Constants Java -> https://howtodoinjava.com/java/basics/java-naming-conventions/

Reply:
great resource thanks for sharing !